fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -295,6 +342,21 @@ if [ ! -e "$fMRIFolder" ] ; then
 fi
 cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
 
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    #  use FSL's fslreorient2std for reorienting the image to match the approximate orientation of the standard template images MNI152
+    #  this makes the single-band processing more robust to registration problems    
+    fslreorient2std "$fMRIFolder"/"$OrigTCSName".nii.gz "$fMRIFolder"/"$OrigTCSName"_orig_reorient.nii.gz
```

Comment:
Looks like this particular script is one that uses the `${FSLDIR}/bin` convention for calling FSL tools, so we should be consistent with that.  Also, don't need the .nii.gz extensions.

Reply:
OK.